tests/dragonfly/connection_test.py

```diff
@@ -1151,3 +1151,22 @@ async def push_pipeline(bad_actor_client, size=1):
             info = await good_client.info()
 
     assert info["dispatch_queue_bytes"] == 0
+
+
+async def test_client_unpause():
```

Comment:
This fails on valkey as well... I need to understand why. 

Interestingly enough, it has no problem with redis-cli (both for us and valkey :thinking: )